docs/spec.md

```diff
@@ -5044,23 +5101,25 @@ def sign(x):
     if is_nan(real(x)) or is_nan(imag(x)): return (NaN, NaN)
     if compare(x, (0.0, 0.0), EQ, FLOAT): return (0.0, 0.0)
     return divide(x, convert(abs(x), type(x)))
+  elif is_quantized_type(x):
+    return dequantize_op_quantize(sign, operand, type(result))`.
```

Comment:
I don't think `dequantize_op_quantize` works well in this function, since it's a scalar function. Let's drop these lines from the diff, and say something like of "For quantized types, performs
`dequantize_op_quantize(sign, operand, type(result))`" a few lines below.

Reply:
done